src/js/components/navbars.js

```diff
@@ -85,48 +86,127 @@
 
   var module = angular.module('mobile-angular-ui.components.navbars', []);
 
-  /**
-   * @directive navbarAbsoluteTop
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned top navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-top">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  var global;
 
   /**
-   * @directive navbarAbsoluteBottom
-   * @restrict C
-   * @description
-   *
-   * Setup absolute positioned bottom navbar.
-   *
-   * ``` html
-   *  <div class="navbar navbar-app navbar-absolute-bottom">
-   *    <!-- ... -->
-   *  </div>
-   * ```
-   */
+  * @directive navController
```

Comment:
Calling a directive controller is really confusing. I fear it would make it hard to understand the purpose of that. Also `nav` is very generic and may mean `navigation` or `navbar`. In this case i believe you meant `navbar`.

We should come out with a better name, or even without a name at all. You can reuse `.navbar ` and make it a directive with `restrict: 'C'`. Its just an idea, still not reached the core part so i don't know what are u gonna use it for.